src/coreclr/tools/aot/Mono.Linker.Tests/TestCasesRunner/ILCompilerTestPInvokePolicy.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Internal.IL;
+using Internal.TypeSystem;
+
+namespace Mono.Linker.Tests.TestCasesRunner
+{
+	internal sealed class ILCompilerTestPInvokePolicy : PInvokeILEmitterConfiguration
+	{
+		public override bool GenerateDirectCall (MethodDesc method, out string? externName)
+		{
+			externName = null;
+			return false;
```

Comment:
This will produce less IL to analyze.

```suggestion
			externName = method.Name;
			return true;
```